scripts/cbsa/parse-metro-areas.ts

```diff
@@ -0,0 +1,92 @@
+import fs from 'fs';
+import path from 'path';
+import { groupBy, values } from 'lodash';
+const promisify = require('util').promisify;
+const csvParse = promisify(require('csv-parse'));
+
+/**
+ * This is a one-off script to parse and format metro areas from the
+ * mar2020.csv file.
+ */
+
+enum StatisticalAreaType {
+  Metro = 'Metropolitan Statistical Area',
+  Micro = 'Micropolitan Statistical Area',
+}
+
+interface CsvRow {
+  cbsaCode: string;
+  metropolitanDivisionCode: string;
+  csaCode: string;
+  cbsaTitle: string;
+  statisticalAreaType: StatisticalAreaType;
+  metropolitanDivisionTitle: string;
+  csaTitle: string;
+  countyEquivalent: string;
+  stateName: string;
+  fipsStateCode: string;
+  fipsCountyCode: string;
+  centralOutlyingCounty: string;
+}
+
+async function main() {
+  const csvText = fs.readFileSync(path.join(__dirname, 'mar2020.csv'));
+  const [, ...rows] = await csvParse(csvText);
+  const items: CsvRow[] = rows
+    .map(parseRow)
+    .filter((item: CsvRow) => item.csaCode);
+
+  const groupedByCbsa = splitBy(items, item => item.cbsaCode);
+  const msaItems = groupedByCbsa
+    .map(formatGroup)
+    .filter(metroArea => metroArea.countyFips.length > 1);
```

Comment:
I'm not 100% sure that we should filter metro areas with only one county out of the list 🤔

Reply:
I'm going to keep metro areas with one county, there is no harm and there are not that many with a single county.